### PR TITLE
It's a new farewell saying you go to eat with you're family(dinner)

### PR DESCRIPTION
**Pull request by Alexxx1407**
literally a good bye saying you go eat dinner with you're family, monika reacts differently depends on you're time. Between, i don't own the two others edits. i only own the "big one"

### CODE REVIEW

**Comment by WiiBii2** [comment]
This looks fantastic! A few grammatical errors, but I see nothing wrong with the code itself! Keep up the good work!

**Comment by Alexxx1407** [comment]
Thanks! i mean i'm not English ... But thanks! so i need to add new things or?

**Comment by WiiBii2** [comment]
I'd run Monika's script through a web-based grammar checker like Grammarly.com just to check for issues like that. It's mostly just nit picky stuff that would sound okay if you were talking out loud, but doesn't work in writing (for instance, saying “you're” instead of “your” and using an incorrect tone).

>            m 2ekd "Are you sure you've had enough rest for it?"

This is the only dialogue that I would deem 'unreadable', but only because it is unclear. I assume you meant to ask the player if they have spent too much time with Monika and if they need a break, in which case I would just say "You should probably take a break!” Like I said though, this is  a very good first contribution, especially for someone who's first language is not English!

**Comment by Alexxx1407** [comment]
I see lemme edit it then, and thanks! i appreciate it. and beside that all good?

**Comment by Alexxx1407** [comment]
Done i edited it, all good!

**Comment by Alexxx1407** [comment]
So will you add it to the main game, or do I need to add some things again?

**Comment by WiiBii2** [comment]
I'm actually not a moderator, so I don't have the privileges to merge the request haha. However, a French translation sounds great! If you open a suggestion under issues, I'm sure the creators would love the idea! If I remember correctly, there has been plenty of talk about translating the game through Renpy's auto translator, but nothing has been done to my knowledge.

**Comment by Alexxx1407** [comment]
Alright I'll try tomorrow!! But can you try to ask if my code will be in the main game ? I'm gonna sleep bye!Sent from Android deviceLe 2 nov. 2021 00:52, WiiBii2 ***@***.***> a écrit :
I'm actually not a moderator, so I don't have the privileges to merge the request haha. However, a French translation sounds great! If you open a suggestion under issues, I'm sure the creators would love the idea! If I remember correctly, there has been plenty of talk about translating the game through Renpy's auto translator, but nothing has been done to my knowledge.

File: Monika After Story/game/script-farewells.rpy [hunk 1]
```diff
@@ -1741,3 +1741,80 @@ label bye_prompt_hangout:
...
+            elif mas_isMoniDis():
+                m 6rkc "You should go have fun, [player]..."
+                m 6dkc "It might make you--{w=1}us--{w=1}happier."
+
+            else:
+                m 6ckc "..."
+
+            return 'quit'
+
+        if session_time < datetime.timedelta(minutes=20):
```
Comment (multimokia) [inline Monika After Story/game/script-farewells.rpy:1782]:
```suggestion
        elif session_time < datetime.timedelta(minutes=20):
```

File: Monika After Story/game/script-farewells.rpy [hunk 2]
```diff
@@ -1741,3 +1741,80 @@ label bye_prompt_hangout:
...
+            m 3hksdlb "I'm just kidding, [mas_get_player_nickname()]."
+            m 2eka "You're so sweet for seeing me even when you have so little time."
+            m 2hub "I just want you to know I really appreciate that!"
+            m 2eka "Eat well! [player], I'm sure that you will enjoy it!"
+            m 2hua "See you when you get back! love you!"
+        elif session_time < datetime.timedelta(hours=1):
+            m 2eua "Alright, thanks for spending some time with me, [player]!"
+            m 2eka "I honestly wish it could have been longer...but you're a busy [guy]."
+            m 2hua "Nothing is more important than you're family?"
+            m 3kublu "Maybe me, ehehehehe~~"
```
Comment (multimokia) [inline Monika After Story/game/script-farewells.rpy:1794]: Consistency: we only have `ehehe` or `ahaha` at those fixed lengths
```suggestion
            m 3kublu "Maybe me, ehehe~"
```

File: Monika After Story/game/script-farewells.rpy [hunk 3]
```diff
@@ -1741,3 +1741,80 @@ label bye_prompt_hangout:
...
+            m 1hka "I'll be waiting for you to come back. Stay safe."
+
+    elif mas_isMoniUpset():
+        m 2esc "Fine, [player]."
+        m "Hopefully you at least eat well."
+        m 2efc "See you."
+
+    elif mas_isMoniDis():
+        m 6rkc "Oh, okay [player]..."
+        m 6lkc "I guess I'll see you after you're diner."
```
Comment (multimokia) [inline Monika After Story/game/script-farewells.rpy:1814]:
```suggestion
        m 6lkc "I guess I'll see you after you're dinner."
```

**Review by multimokia — CHANGES_REQUESTED** [review]

**Comment by multimokia** [comment]
please rebase your PR. All PRs should be targetting `content`

**Comment by KiddTheManiac** [comment]
I'm seeing a lot of weird grammatical errors, this might need some extra proofreading? Like the "you're dinner" and "you're family". Both of those should just be "your", I believe.

**Comment by Alexxx1407** [comment]
Don't worry guys, i will fix everything tomorrow, i need to sleep for now, good night!!Sent from Android deviceLe 2 nov. 2021 02:02, KiddTheManiac ***@***.***> a écrit :
I'm seeing a lot of weird grammatical errors, this might need some extra proofreading? Like the "you're dinner" and "you're family". Both of those should just be "your", I believe.

**Comment by multimokia** [comment]
Going to close this as it seems a little unnecessary given we already have an `I'm going to eat` farewell, this can more or less fall under that same umbrella.

**Pull request closed without merging**